crates/runtime/src/mmap.rs

```diff
@@ -19,7 +16,7 @@ fn round_up_to_page_size(size: usize, page_size: usize) -> usize {
 /// and initially-zeroed memory and a length.
 #[derive(Debug)]
 pub struct Mmap {
-    ptr: *mut u8,
+    offset: usize,
```

Comment:
Naming nit: I think `offset` is a confusing name because it's not relative to something else.  It's still the pointer, just stored as a `usize`, right?

Reply:
It is yeah, I figured though that `ptr` wasn't so appropriate because it wasn't actually a pointer. I'll leave though as `ptr` and write in a comment as to why it's a `usize`